### PR TITLE
Public interop devnet details

### PR DESCRIPTION
**Pull request by sbvegan**
**Description**

- Chain details on devnet-0 and devnet-1
- Information on how to make an ether deposit into your account on the L2
- Information on the new predeploys

fixes: https://github.com/ethereum-optimism/devrel/issues/378

### CODE REVIEW

File: pages/stack/interop/devnet.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+description: Details on the public interoperability devnets.
+---
+
+import { Callout, Tabs, Steps } from 'nextra/components'
+
+# Interop devnet
+
+The Interop Devnet is a temporary public network of two OP Stack Sepolia instances that supports SuperERC20 tokens, native cross-chain messaging, and cross-chain ETH transfers. This network will be deprecated once the next devnet is released.
+
+## Interop Devnet 0
```
Comment (cpengilly) [inline pages/stack/interop/devnet.mdx:13]:
```suggestion
## Interop devnet 0
```

File: pages/stack/interop/devnet.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+| --------------------------- | ---------------------------------------------------------------------------------------------------- |
+| Network Name                | `Interop Devnet 0`                                                                                   |
+| Chain ID                    | `11473209`                                                                                           |
+| Currency Symbol<sup>1</sup> | ETH                                                                                                  |
+| Explorer                    | [https://optimism-interop-devnet-0.blockscout.com](https://optimism-interop-devnet-0.blockscout.com) |
+| Public RPC URL              | [https://interop-devnet-0.optimism.io](https://interop-devnet-0.optimism.io)                         |
+| Sequencer URL               | [https://interop-devnet-0.optimism.io](https://interop-devnet-0.optimism.io)                         |
+| OptimismPortal<sup>2</sup>  | `0xb081417d0578d70eb6bee7bfa646ef8c283eb35a`                                                         |
+
+## Interop Devnet 1
```
Comment (cpengilly) [inline pages/stack/interop/devnet.mdx:25]:
```suggestion
## Interop devnet 1
```

File: pages/stack/interop/devnet.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+<Steps>
+  ### Get Sepolia ETH
+
+  You can utilize the [Superchain Faucet](https://console.optimism.io/faucet) to get ether on Sepolia.
+
+  ### Send the Sepolia ETH to the devnet
+
+  You can send ether directly to the `OptimismPortal` address and it will go to the same sender address on the devnet.
+
+  ### Wait
```
Comment (cpengilly) [inline pages/stack/interop/devnet.mdx:51]: could we make this heading more descriptive? something like: 
```suggestion
  ### Wait for bridging to complete
```

File: pages/stack/interop/explainer.mdx [hunk 4]
```diff
@@ -40,6 +40,43 @@ Chain operators can add or remove chains from the dependency set through the `Sy
...
+
+### SuperchainERC20Bridge
+
+The `SuperchainERC20Bridge` is an abstraction on top of the `L2toL2CrossDomainMessenger` that facilitates token bridging using interop. It has mint and burn rights over `SuperchainERC20` tokens, as described in the [token bridging spec](https://specs.optimism.io/interop/token-bridging.html).
+
+*   **Address:** `0x4200000000000000000000000000000000000028`
+*   **Specs:** [SuperchainERC20Bridge](https://specs.optimism.io/interop/predeploys.html#superchainerc20bridge)
 ## Considerations
 Chain operators will need to run additional infrastructure to be part of the interoperable set. 
 *   The Superchain-backend service, `op-supervisor`, will be a requirement for running an OP Stack chain that has interop enabled.
```
Comment (cpengilly) [inline pages/stack/interop/explainer.mdx:84]: we now have this page/link available. yay!
```suggestion
*   The Superchain-backend service, [`op-supervisor`](op-supervisor), will be a requirement for running an OP Stack chain that has interop enabled. 
```

**Review by cpengilly — APPROVED** [review]

File: pages/stack/interop/devnet.mdx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+title: Interop devnet
+lang: en-US
+description: Details on the public interoperability devnets.
+---
+
+import { Callout, Tabs, Steps } from 'nextra/components'
+
+# Interop devnet
+
+The Interop devnet is a temporary public network of two OP Stack Sepolia instances that supports SuperERC20 tokens, native cross-chain messaging, and cross-chain ETH transfers. This network will be deprecated once the next devnet is released.
```
Comment (zainbacchus) [inline pages/stack/interop/devnet.mdx:11]: As we iterate on Superchain interop, these networks will be deprecated once the next devnets are released.

**Review by zainbacchus — APPROVED** [review]
looks good see one nit comment

**Pull request merged**